packages/web/components/Setup/SetupProfilePicture.tsx

```diff
@@ -0,0 +1,120 @@
+  const [preview, setPreview] = useState<string | null>();
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  return (
+    <ProfileWizardPane
+      {...{ field }}
+      title="Profile Picture"
+      prompt="Upload an image that will make you instantly recognizable."
+    >
+      {({ register, setter, current }: WizardPaneCallbackProps) => {
+        const { ref: registerRef } = register(field, {});
+
+        const handleUpload = async (e: React.ChangeEvent<HTMLInputElement>) => {
+          const imageFile = e.target.files?.[0];
+          if (!imageFile) return;
+
+          const img = new Image();
+          img.src = URL.createObjectURL(imageFile);
+          img.onload = () => {
+            const imgData = {
+              file: imageFile,
+              width: img.width,
+              height: img.height,
+            };
+            setImage(imgData);
+            setPreview(URL.createObjectURL(imageFile));
+            // need to pass the height and width to the setter in order to pass it to ceramic
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            setter(imgData);
+          };
+        };
+        const handleClick = () => {
+          inputRef.current?.click();
+          inputRef.current?.focus();
+        };
+        const handleRemove = () => {
+          setImage(null);
+          setter('');
+          setPreview(null);
+        };
+
+        return (
+          <VStack mt={5}>
+            <Center mb="6">
+              <ChakraImage
+                height="200px"
+                width="200px"
+                objectFit="cover"
+                borderRadius="full"
+                src={preview || current || OctoAvatar.src}
+              />
+            </Center>
+            <Input
+              type="file"
+              onChange={handleUpload}
+              name="profileImageURL"
+              accept="image/png, image/jpeg, image/gif"
+              ref={(ref) => {
+                ref?.focus();
+                registerRef(ref);
+                inputRef.current = ref;
+              }}
+              display="none"
+            />
+            <Button
+              colorScheme="purple"
+              px={8}
+              letterSpacing="0.1em"
+              size="md"
+              fontSize="sm"
+              bg="purple.400"
+              color="white"
+              mt={{ base: 2, md: 6 }}
+              onClick={handleClick}
+            >
+              Upload image
+            </Button>
+            <Flex height={{ base: '3rem', md: '4rem' }}>
+              {current || preview ? (
+                <Button
+                  mt={{ base: 2 }}
+                  color="#EB5757"
+                  variant="ghost"
+                  size="md"
+                  fontSize="sm"
+                  onClick={handleRemove}
+                >
+                  Remove Image
+                </Button>
+              ) : null}
```

Comment:
```suggestion
              {current || preview && (
                <Button
                  mt={2}
                  color="#EB5757"
                  variant="ghost"
                  size="md"
                  fontSize="sm"
                  onClick={handleRemove}
                >
                  Remove Image
                </Button>
              )}
```

Reply:
I tried this, but doesn't work as you expect, it will return the preview, because preview is truthy, and the second part of && statement doesn't get reached.

Reply:
You probably just need parens, but it's not worth revisiting.

Reply:
> 

Yes, probably, and my auto formatter was adding parens automatically. Agreed, not worth revisiting.